docker-compose.yml

```diff
@@ -1,3 +1,8 @@
+# dotenv gem doesn't always do the right thing with blank env vars, so we make sure these get set to something
+x-app-env: &app-env
```

Comment:
@mark-dce adding it to be interpreted by the docker-compose means we can still override the value. both:
`HYRAX_FLEXIBLE=true sc up` and 

```
sc sh
HYRAX_FLEXIBLE=true ./bin/rails c
```
will work properly with this addition.